pkg/rpm/build.py

```diff
@@ -1,50 +1,56 @@
 #! /bin/env python
 from __future__ import print_function
-import sys
+
+import argparse
 import os
+import sys
 import tarfile
-import argparse
-from os.path import dirname, join, abspath
+from os.path import abspath, dirname, join
 from shutil import copy
 from subprocess import check_call
 
-parser = argparse.ArgumentParser(
-        description='Build salt rpms',
-        )
-parser.add_argument('buildid',
-        help='The build id to use i.e. the bit after the salt version in the package name',
-        )
+import salt.version
```

Comment:
This `salt.version` import should probably go back to where it was, see the `sys.path.append` before it?